seatunnel-connectors-v2/connector-iceberg/src/main/java/org/apache/seatunnel/connectors/seatunnel/iceberg/config/SinkConfig.java

```diff
@@ -109,13 +109,20 @@ public class SinkConfig extends CommonConfig {
                     .defaultValue(DataSaveMode.APPEND_DATA)
                     .withDescription("data save mode");
 
+    public static final Option<String> TABLES_DEFAULT_COMMIT_BRANCH =
```

Comment:
add to this file?
https://github.com/apache/seatunnel/blob/dev/seatunnel-connectors-v2/connector-iceberg/src/main/java/org/apache/seatunnel/connectors/seatunnel/iceberg/sink/IcebergSinkFactory.java#L57

Reply:
> add to this file? https://github.com/apache/seatunnel/blob/dev/seatunnel-connectors-v2/connector-iceberg/src/main/java/org/apache/seatunnel/connectors/seatunnel/iceberg/sink/IcebergSinkFactory.java#L57

@hailin0 done